arbitrum-client/src/client/contract_interaction.rs

```diff
@@ -111,6 +114,8 @@ impl ArbitrumClient {
     ///
     /// Awaits until the transaction is confirmed on-chain
     #[instrument(skip_all, err, fields(
+        proof_calldata,
+        statement_calldata,
```

Comment:
do we really want to log all calldata? That's quite a lot of data to be sending to DD

Reply:
Plus if it's a hex string that's almost entirely useless in a trace except to check it against another hex string. That case is unlikely to come up (under working circumstances all calldata is mostly different). I'd say dumping calldata into DD doesn't warrant the benefit it gives

Reply:
I did use this when debugging the `verification failed` issue from before, I can decode this calldata in a unit test that uses it to invoke contract code. This is generally something I could get from a transaction receipt, but of course that's only when the transaction succeeds.

My rationale here is that the testing we're doing is surfacing issues that can be hard to reproduce, so it's quite helpful to be equipped w/ test data when an issue is observed, and I figure we can eat the DD bill for a while.

But that said, this does feel clunky, and is a lot of data that we do pay for - generally the observed issues are reproducible if you just give the load testing a bit of time to snag them again.

So I'm ok with removing right now